src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/dcoff.cpp

```diff
@@ -105,7 +114,14 @@ void apply_remap(Subgraph& fcall, const ClosureRemap& m) {
 
         auto scale_iter = m.scale_remap.find(i);
         new_scales.push_back(scale_iter != m.scale_remap.end() ? fcall._closure[scale_iter->second] : ov::Tensor());
-        new_zerops.push_back(m.zero_points[i]);
+        // Check for asymmetric zero points and add them to new_zerops
+        auto asymm_zerop_iter = m.asymm_zero_points.find(i);
+        if (asymm_zerop_iter != m.asymm_zero_points.end()) {
+            new_zerops.push_back(fcall._closure[asymm_zerop_iter->second]);
+        } else {
+            // If there is no asymmetric zero point, use the existing symmetric zero point
+            new_zerops.push_back(m.zero_points[i]);
+        }
```

Comment:
as you do the `push_back` regardless,

```suggestion
        const auto &zerop = asymm_zerop_iter != m.asymm_zero_points.end() ? fcall._closure[asymm_zerop_iter->second] : m.zero_points[i];
        new_zerops.push_back(zerop);
```

Reply:
Changed!